api/types/accesslist/accesslist_test.go

```diff
@@ -393,3 +395,11 @@ func TestAccessList_setInitialReviewDate(t *testing.T) {
 		})
 	}
 }
+
+func TestAccessListClone(t *testing.T) {
```

Comment:
Make sure after adding any new field the test will fails and a developer will need to align the Clone method.